snippets/csharp/System.Linq/Enumerable/Comparers/EncapsulatedComparer.cs

```diff
@@ -136,13 +151,15 @@ lemon 12
 
         //<SequenceEqual>
 
-        ProductA[] storeA = { new ProductA { Name = "apple", Code = 9 },
-                               new ProductA { Name = "orange", Code = 4 } };
+        ProductA[] storeA = [
+            new ("apple", 9 ),
+            new ("orange", 4 ) ];
 
-        ProductA[] storeB = { new ProductA { Name = "apple", Code = 9 },
-                               new ProductA { Name = "orange", Code = 4 } };
+        ProductA[] storeB = [
+            new ("apple", 9 ),
+            new ("orange", 4 ) ];
 
-        bool equalAB = storeA.SequenceEqual(storeB);
+        var equalAB = storeA.SequenceEqual(storeB);
```

Comment:
Here, I'd prefer an explicit `bool` type. Without a rollover and intellisense, it's better to show the return type in text.